templates/observation.html

```diff
@@ -851,6 +846,63 @@ <h5 class="modal-title" id="confirmModalLabel">Reset media</h5>
         selection: false
       });
 
+      const canvasHeight = 332;
+      const canvasWidth = 530;
+
+      if (!is_touch_device) {
+        stage.on('mouse:wheel', function(opt) {
+          var delta = opt.e.deltaY * -1;
+          var pointer = stage.getPointer(opt.e);
+          var zoom = stage.getZoom();
+          zoom = zoom + delta / 200;
+
+          if (zoom > 10) {
```

Comment:
you can use max/min  functions instead. e.g. `zoom = max(10, min(1, zoom))`. Please check js syntax for that